spec/gameSpec.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+describe('Game', function() {
+  let game
+
+  beforeEach(function() {
+    game = new Game();
+    game.beginFrame();
+  })
+
+  it('begins a new frame',function() {
+    expect(game.turn).toEqual(1)
+  })
+
+  it('has an empty scorecard at first', function() {
+    expect(game.scorecard).toEqual([])
+  })
+
+  it('adds the users points to each respective frame', function() {
+    game.throwBall(7);
+    game.addToScorecard();
+    expect(game.scorecard[0]).toEqual([ 7 ])
```

Comment:
As Adam pointed out in his comment above, some of your tests (like this one) are testing state rather than implementation. It might be useful to revisit [this practical](https://github.com/makersacademy/skills-workshops/blob/master/practicals/testing/behaviour_not_state.md) on the topic. The "How do I test behaviour?" section in particular shows an example of how to think about the difference between testing state and behaviour and derive an appropriate test case. 

In this case you are directly accessing the scorecard array which means that later if you change your mind about how to store your scorecard, you're going to have to update your tests. Perhaps your `Game` and/or `Frame` classes are missing some method(s) to allow a user of the class to easily find out what's on the scorecard for a particular frame?

Reply:
I'll check that practical out. That all makes sense so I will give this some thought and practice, thank you.